contracts/tests/market_makers/test_calc_token_count.py

```diff
@@ -11,41 +12,55 @@ def __init__(self, *args, **kwargs):
         self.centralized_oracle_factory = self.create_contract('Oracles/CentralizedOracleFactory.sol')
         self.market_factory = self.create_contract('Markets/DefaultMarketFactory.sol')
         self.lmsr = self.create_contract('MarketMakers/LMSRMarketMaker.sol', libraries={'Math': self.math})
-        self.ether_token = self.create_contract('Tokens/EtherToken.sol')
         self.token_abi = self.create_abi('Tokens/AbstractToken.sol')
         self.market_abi = self.create_abi('Markets/DefaultMarket.sol')
         self.event_abi = self.create_abi('Events/AbstractEvent.sol')
 
     @staticmethod
-    def calc_token_count(token_count, token_index, token_distribution, funding):
-        _max = max(token_distribution)
-        token_distribution = [_max - x for x in token_distribution]
-        b = funding / float(math.log(len(token_distribution)))
-        return b * math.log(
-            sum([math.exp(share_count / b + token_count / b) for share_count in token_distribution]) -
-            sum([math.exp(share_count / b) for index, share_count in enumerate(token_distribution) if index != token_index])
-        ) - token_distribution[token_index]
```

Comment:
Consider putting back, but at least implement it down the line and test with gnosis.js